prow/github/report/report.go

```diff
@@ -156,7 +159,16 @@ func Report(ghc GithubClient, reportTemplate *template.Template, pj prowapi.Prow
 	if len(refs.Pulls) == 0 {
 		return nil
 	}
-
+	icomm, err := ghc.ListPRCommits(refs.Org, refs.Repo, refs.Pulls[0].Number)
```

Comment:
^^ there's already a `ListIssueComments` below which does the same thing :-)

Reply:
hm, but `ListIssueComments` is for comments? i use this call (to `ListPRCommits`) to get the commits for a PR job at the time when this job reaches `Report()`.

Reply:
🤦‍♂️ sorry, I read it as `ListPRComments` and got confused